app/src/androidTest/java/com/sdp/movemeet/view/home/HomeScreenActivityLoggedTest.java

```diff
@@ -77,4 +85,23 @@ public void recordButtonHasCorrectIntent() {
         onView(withId(R.id.recordRunButton)).perform(click());
         intended(hasComponent(GPSRecordingActivity.class.getName()));
     }
+
+    public static ViewAction forceDoubleClick() {
```

Comment:
I don't think this method is used in this class, shouldn't it be deleted ? At first, I thought you made it static to use it in other classes, but from what I see, you're redefining it everytime.

Reply:
It's true, I had put it back because of a problem while clicking but the problem came from my phone and I forgot to remove it thanks!